src/Nonlinear/parse.jl

```diff
@@ -36,6 +36,58 @@ function parse_expression(::Model, ::Expression, x::Any, ::Int)
     )
 end
 
+function parse_expression(
+    data::Model,
+    expr::Expression,
+    x::MOI.ScalarNonlinearFunction,
+    parent::Int,
```

Comment:
What's the parent argument? Please document.

Reply:
Documented to some extent in
https://github.com/jump-dev/MathOptInterface.jl/blob/978418089e5e24534f192e378bbd6c4b763707dd/src/Nonlinear/parse.jl#L18-L32